File: app/src/main/java/com/forkmang/activity/FaceLoginPermission.java
package com.forkmang.activity;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.widget.Button;

import com.forkmang.R;

public class FaceLoginPermission extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_faceloginpermission);
        Button Btn_Enable = findViewById(R.id.btn_enable);

        Btn_Enable.setOnClickListener(v -> {
            showAlertView();
        });

    }

    private void showAlertView()
    {
        final AlertDialog.Builder alertDialog = new AlertDialog.Builder(FaceLoginPermission.this);
        LayoutInflater inflater = (LayoutInflater) getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        final View dialogView = inflater.inflate(R.layout.conform_tablereserve_view, null);
        alertDialog.setView(dialogView);
        alertDialog.setCancelable(true);
        final AlertDialog dialog = alertDialog.create();
        Button Btn_Cancel,Btn_Yes ;

        Btn_Cancel=dialogView.findViewById(R.id.btn_cancel);
        Btn_Yes=dialogView.findViewById(R.id.btn_yes);

        Btn_Cancel.setOnClickListener(v -> {
            dialog.dismiss();
        });

        Btn_Yes.setOnClickListener(v -> {
            dialog.dismiss();
            /*final Intent mainIntent = new Intent(FaceLoginPermission.this, FaceLogin.class);
            startActivity(mainIntent);
            finish();*/

        });

        dialog.show();

    }


}

File: app/src/main/java/com/forkmang/activity/FaceLoginPermission.kt
package com.forkmang.activity

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import com.forkmang.R

class FaceLoginPermission : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_faceloginpermission)

        val btnEnable: Button = findViewById(R.id.btn_enable)
        btnEnable.setOnClickListener { showAlertView() }
    }

    private fun showAlertView() {
        val alertDialog: AlertDialog.Builder = AlertDialog.Builder(this@FaceLoginPermission)
        val inflater: LayoutInflater = getSystemService(LAYOUT_INFLATER_SERVICE) as LayoutInflater
        val dialogView: View = inflater.inflate(R.layout.conform_tablereserve_view, null)
        alertDialog.setView(dialogView)
        alertDialog.setCancelable(true)
        val dialog: AlertDialog = alertDialog.create()
        val btnCancel: Button = dialogView.findViewById(R.id.btn_cancel)
        val btnYes: Button = dialogView.findViewById(R.id.btn_yes)
        btnCancel.setOnClickListener { dialog.dismiss() }
        btnYes.setOnClickListener { dialog.dismiss() }
        dialog.show()
    }
}